.evg.yml

```
@@ -581,9 +632,12 @@ functions:
           # Tag triggered runs are releases and the version is set in the tag.
           # Other runs are snapshot builds (periodic builds or patches)
           if [[ "${triggered_by_git_tag}" != "" ]]; then
-            export MDBJDBC_VER=$(echo ${triggered_by_git_tag} | sed s/v//)
+            # tag should be formatted as 'v<JDBC major>.<minor>.<patch>-libv<libmongosqltranslate major>.<minor>.<patch>
+            export MDBJDBC_VER=$(echo ${triggered_by_git_tag} | sed -e 's/\-libv.*//' | sed s/v// )
+            export LIBMONGOSQLTRANSLATE_VER=$(echo ${triggered_by_git_tag} | sed -e 's/v.*\-libv//')
```

You'd have less potential for problems using cut to retrieve the part of the tag you want and then remove the prefix. `sed -e` is not super cross-platform friendly, using cut is safer.
```suggestion
            export MDBJDBC_VER=$(echo ${triggered_by_git_tag} | cut -d '-' -f 1 | sed s/v// )
            export LIBMONGOSQLTRANSLATE_VER=$(echo ${triggered_by_git_tag} | cut -d '-' -f 2 | sed s/libv//)
```